### PR TITLE
Use pyprint instead of print

### PR DESCRIPTION
**Pull request by Dhwaniartefact**

### CODE REVIEW

File: tests/test_fixity.py [hunk 1]
```diff
@@ -462,6 +462,5 @@ def test_main_handles_exceptions_if_scanall_fails(_get, monkeypatch, capsys):
 
     captured = capsys.readouterr()
     assert (
-        captured.out.strip()
-        == f"Internal error encountered while scanning AIP {aip_id} (StorageServiceError)"
-    )
+        f"Internal error encountered while scanning AIP {aip_id} (StorageServiceError)\n"
+    ) in captured.out
```
Comment (replaceafill) [inline tests/test_fixity.py]: Why is this change necessary?

File: fixity/utils.py [hunk 2]
```diff
@@ -33,3 +34,10 @@ def check_valid_uuid(uuid):
 
 def utcnow():
     return datetime.now(timezone.utc)
+
+
+def pyprint(message):
+    if message.startswith("Internal error encountered"):
```
Comment (replaceafill) [inline fixity/utils.py]: Why does this depend on the contents of the `message`? Suppose there's new code that adds another message that starts with the same substring but needs to send the output to standard error instead.

Could this be implemented with a keyword argument like the [`Job.pyprint` method](https://github.com/artefactual/archivematica/blob/bba53bad33c0ce0a6453fedc3f5b62bda303faa3/src/MCPClient/lib/client/job.py#L111-L122) does in the MCPClient?

**Review by replaceafill — CHANGES_REQUESTED** [review]

File: tests/test_fixity.py (hunk 1 above)
Comment (Dhwaniartefact) [reply tests/test_fixity.py]: just to keep uniformity same as above assertions for `test_scanall_handles_exceptions`

Comment (replaceafill) [reply tests/test_fixity.py]: I see. Good thinking, but maybe we could do the opposite: change the `in` comparison in `test_scanall_handles_exceptions` to be an equality comparison with `==`.

We want exact matches as much as possible, don't you think?

**Review by replaceafill — APPROVED** [review]
